packages/twenty-server/src/utils/generate-front-config.ts

```diff
@@ -1,17 +1,14 @@
-import { ConfigService } from '@nestjs/config';
-
 import * as fs from 'fs';
 import * as path from 'path';
 
-import { EnvironmentService } from 'src/engine/integrations/environment/environment.service';
-
-const environmentService = new EnvironmentService(new ConfigService());
+import { config } from 'dotenv';
+config();
```

Comment:
Tbh I'm not 100% sure this is required but it was there in other similar raw files (e.g. core.datasource.ts)

Reply:
In main.ts it isn't there but I'm guessing is done by a depedency ; tI ried changing the port and it worked